completion/available/github-cli.completion.bash

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+cite "about-completion"
+about-completion "GitHub CLI completion"
+
+if _command_exists gh; then
+  if _command_exists brew; then
+    echo "You don't need github-cli completion enabled if you have system completion enabled"
+  fi
```

Comment:
@nwinkler thoughts on including this message? Not sure how I feel about it, but it would be nice to provide some guidance to users. Couple of things that might make it better:

- Some sort of standard warning message for bash-it that could be turned off with `BASH_IT_WARN=off` (or maybe its a log and we use log levels?)
- Having a BASH_IT_HOMEBREW that does `_command exists brew` once so we can use it without having to check each time
- Some sort of detection and only show this if system is enabled. Something like `if _completion_enabled "system"; then`

Don't really have much bandwidth to tackle any of those though. :-( Happy to leave some `TODO` notes with a reference back to this discussion or just simply remove it if ya think it's too much.

Reply:
I'm not in favour of including `echo` messages in Bash-it components, since they cause unwanted noise.

The discussion in #1501 has some of the ideas that you mention above. We talked about potentially adding a logging system and a `bash-it doctor` command that prints log messages when run.

Until then we should probably remove the `echo` message...

Thanks for the other ideas, they are great!

Reply:
Replaced the `echo` with `_log_warning` in #1670